src/payment/checkout/payment-form/PaymentForm.jsx

```diff
@@ -208,7 +208,9 @@ export class PaymentFormComponent extends React.Component {
                   <StatefulButton
                     type="submit"
                     id="placeOrderButton"
-                    className="btn btn-primary btn-lg btn-block"
+                    variant="primary"
+                    size="lg"
+                    block
```

Comment:
is this a parameter?

Reply:
New way - we're not using classNames in Buttons and StatefulButtons anymore https://edx.github.io/paragon/components/statefulbutton